packages/table/src/react/index.js

```diff
@@ -1,71 +1,45 @@
-import { vars as drawerVars } from '@pluralsight/ps-design-system-drawer'
 import * as glamor from 'glamor'
-import Icon from '@pluralsight/ps-design-system-icon/react'
-import { defaultName as themeDefaultName } from '@pluralsight/ps-design-system-theme/react'
 import PropTypes from 'prop-types'
 import React from 'react'
 
+import { vars as drawerVars } from '@pluralsight/ps-design-system-drawer'
```

Comment:
I would use `/vars` in the import instead in order to not get the whole Drawer code included here. (current untreeshakeable)